requirements.txt

```diff
@@ -8,6 +8,10 @@ torch
 num2words==0.5.10
 word2number==1.1
 
+# for SuspectingParaphraser
```

Comment:
Instead of directly putting any new package specific to your transformation, better to create a separate 'requirements.txt' file under your transformation folder.